packages/web-components/src/components/card-group/__stories__/card-group.stories.ts

```diff
@@ -131,6 +132,47 @@ withImagesAndCTA.story = {
   },
 };
 
+export const withMixedMedia = () => {
+  return html`
+    <dds-video-cta-container>
+      <dds-card-group>
+        <dds-card-group-item cta-type="video" href="1_9h94wo6b">
+          <dds-card-cta-footer cta-type="video" slot="footer" href="1_9h94wo6b"> </dds-card-cta-footer>
+        </dds-card-group-item>
+        <dds-card-group-item href="https://example.com">
+          <dds-card-cta-image slot="image" alt="Image alt text" default-src="${imgXlg4x3}"> </dds-card-cta-image>
+          Image copy
+          <dds-card-footer slot="footer">
+            ${ArrowRight20({ slot: 'icon' })}
+          </dds-card-footer>
```

Comment:
Might be unrelated to this issue, but I am curious shouldn't this be dictated by a `cta-type="local"` instead of hard coding the arrow here? I didn't even know we could do this. This seems like too much flexibility here, but maybe I'm missed an update somewhere.

Reply:
Thanks and okay makes sense!